acceptor.go

```diff
@@ -69,10 +70,11 @@ func (w *WebSocketAcceptor) Spec(subProtocol string) (serializers.Serializer, er
 	return serializer, nil
 }
 
-func (w *WebSocketAcceptor) Accept(conn net.Conn) (BaseSession, error) {
+func (w *WebSocketAcceptor) Accept(conn net.Conn, keepaliveInterval time.Duration,
```

Comment:
these two parameters should be moved into `WebSocketServerConfig` and Accept should accept that.